apps/rxjs-web-demo-e2e/cypress.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I don't see a benefit in using e2e tests here. I think everything should be testable without a ui. So I'd appreciate if you could remove the demo and e2e folders

Reply:
Sure can, although it I could suggest a demo site is always good, showing some basic usage (can even be deployed to gihub pages) but if your sure it's not needed I can remove it

Reply:
I'd prefer adding it when necessary and for now I can not imagine a proper use case. I just want the repo as slim as possible to make it easy to contribute